app.js

```diff
@@ -29,7 +29,7 @@ const db = require('./lib/db');
 
 const fruits = require('./lib/routes/fruits');
 
-app.use(bodyParser.json());
+app.use(bodyParser.text({ type: 'application/json' }));
```

Comment:
why do you need to do this.  isn't this just another way of writing `bodyParser.json()`?

Reply:
Good point. I look at on it.